index.html

```diff
@@ -0,0 +1,137 @@
+            </div>
+        </nav>
+        <div class="headline mt-4">
+            <div class="container hl-wrapper">
+                <p>Sharing the knowledge</p>
+                <h1>WebDev Summit 2022</h1>
+                <div class="hl-box">
+                    <p>A celebration to Web Development. 4 days of fast-paced seminars, lectures, exhibitions and many
+                        more activities. Join us in this awesome summit where more than 80 countries come together to share their knowledge on web development</p>
+                </div>
+                <p class="date">2022.12.12(MON) ~ 15(THU)</p>
+                <p class="date-text">@ National Center of New Technologies </p>
+            </div>
+        </div>
+    </header>
+    <main>
+        <section class="main-program pt-4 pb-4">
+            <h2 class="text-center text-white">Main Program</h2>
+            <hr class="section-hr mx-auto mb-4">
+            <div class="program-wrapper container d-flex flex-column flex-md-row">
+                <div class="activity d-flex flex-md-column align-items-center justify-content-between justify-content-md-around">
+                    <img src="img/lecture.png" alt="" class="activity-img">
+                    <h3 class="w-25">Lecture</h3>
+                    <p class="w-50 text-white">Lorem ipsum dolor sit amet consectetur adipisi elit, natus ea id.</p>
+                </div>
+                <div class="activity d-flex flex-md-column align-items-center justify-content-between justify-content-md-around">
+                    <img src="img/exhibition.png" alt="" class="activity-img">
+                    <h3 class="w-25">Exhibition</h3>
+                    <p class="w-50 text-white">Fugiat quaerat tempore repudiandae expedita doloribus, sit consectetur.
+                    </p>
+                </div>
+                <div class="activity d-flex flex-md-column align-items-center justify-content-between justify-content-md-around">
+                    <img src="img/forum.png" alt="" class="activity-img">
+                    <h3 class="w-25">Forum</h3>
+                    <p class="w-50 text-white">Voluptatem repellat esse neque officiis, vel obcaecati voluptate ipsa.
+                    </p>
+                </div>
+                <div class="activity d-flex flex-md-column align-items-center justify-content-between justify-content-md-around">
+                    <img src="img/workshop.png" alt="" class="activity-img">
+                    <h3 class="w-25">Workshop</h3>
+                    <p class="w-50 text-white">Accusamus sed vel harum totam eveniet, natus ea id perferendis ab
+                        aspernatur.</p>
+                </div>
+                <div class="activity d-flex flex-md-column align-items-center justify-content-between justify-content-md-around">
+                    <img src="img/ignite.png" alt="" class="activity-img">
```

Comment:
Kindly, the Alt attributes should contain a description of your image, that will greatly help the accessibility of your website.